README.md

```diff
@@ -26,3 +26,21 @@ Debian build facilities are integrated into the repo at `debian/`.
 3. Install the resulting `jellyfin*.deb` file on your system.
 
 A huge thanks to Carlos Hernandez who created the Debian build configuration for Emby 3.1.1.
+
+### Windows (64 bit)
+A pre-built windows installer will be available at [The JellyFin Repository](https://repo.jellyfin.org/).
+
+1. Install the dotnet core SDK 2.1 from [Microsoft's Webpage](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-2.1.500-windows-x64-installer)
```

Comment:
Linking to the general download page (https://dotnet.microsoft.com/download) means this link won't become out-of-date later down the line, and the SDK link is obvious on this landing page.

Reply:
Agreed, I did a similar thing in the Debian sections.

Reply:
Link for reference: https://dotnet.microsoft.com/download/dotnet-core/2.1